README.md

```diff
@@ -0,0 +1,10 @@
+# JARED DUNN AI
```

Comment:
It's always cool to have a README file for your projects so people know how to use them 😁